azure_functions_worker/dispatcher.py

```diff
@@ -459,6 +461,7 @@ async def _handle__function_environment_reload_request(self, req):
         try:
             logger.info('Received FunctionEnvironmentReloadRequest, '
                         'request ID: %s', self.request_id)
+            enable_debug_logging_recommendation()
```

Comment:
@vrdmr Should we pass this along with the above 'Received FunctionEnvironmentReloadRequest' log, so we have only one grpc call here? Same with the worker init request

Reply:
During the init, it won't matter much as grpc overhead for only one call is very low. It is when that call is repeated multiple times, then it'll start hampering us.

Also, as this is something that we are expecting customers to observe and take action if necessary, I think we shouldn't mix it with the other logs.

I would recommend keeping as it is.